osh/client/commands/cmd_mock_build.py

```diff
@@ -15,6 +16,16 @@ def options(self):
 
         # this option cannot be used for diff-build tasks
         add_tarball_option(self.parser)
+        add_dist_git_url_option(self.parser)
+
+    def prepare_task_options(self, args, kwargs):
+        # Check if exactly one of SRPM, NVR, or dist-git URL is provided
+        if sum([bool(args), bool(kwargs.get("nvr")), bool(kwargs.get("git_url"))]) != 1:
```

Comment:
Is it possible to define two or more values?

Reply:
Prior to this change, the expected sole positional argument is either srpm or NVR. I don't think 2 or more values will be supported.

Reply:
The expected behavior of being provided with more than 1 values should be that an error being raised.